build/source/engine/soilLiqFlx.f90

```diff
@@ -194,6 +192,7 @@ subroutine soilLiqFlx(&
  ! local variables: general
  character(LEN=256)                  :: cmessage                     ! error message of downwind routine
  integer(i4b)                        :: ibeg,iend                    ! start and end indices of the soil layers in concatanated snow-soil vector
+ logical(lgt),parameter              :: deriv_desired=.true.         ! flag indicating if derivatives are desired
```

Comment:
I believe this is the first use of default function argument values in the code. Is that correct? 

If so, with default arguments we trade readability (user sees less text while reading the code) for clarity (the fact that an argument to the function is available at all becomes hidden). We don't currently have easily accessible function-level documentation that a user can check to learn more about individual functions. In this light, is this trade-off worth it?